virttest/guest_agent.py

```diff
@@ -725,3 +725,42 @@ def fsthaw(self, check_status=True):
                     raise
             return ret
         return -1
+
+    def openfile(self, args):
+        """
+        :params args: It includes file's 'path' and 'mode' option.
+         path = /tmp/agentfile and mode = "w+".
+         if file doesn't exist,mode w+ will create a new file
+        """
+        cmd = "guest-file-open"
+        if self.check_has_command(cmd):
+            return self.cmd(cmd, args, debug=True, success_resp=True)
```

Comment:
The default value of `debug` and `success_resp` are `True`, so do not need to set them explicitly.
https://github.com/avocado-framework/avocado-vt/blob/master/virttest/guest_agent.py#L402

Reply:
@luckyh Thanks,  I have updated related code.